<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
   <item>
    <layout class="QVBoxLayout" name="sidebar_layout_courseinfo">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="main_page_button_courseinfo">
       <property name="font">
        <font>
         <family>Noto Sans CJK TC</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="login_history_button_courseinfo">
       <property name="font">
        <font>
         <family>Noto Sans CJK HK</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Login History</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="logout_button_courseinfo">
       <property name="font">
        <font>
         <family>Noto Sans CJK HK</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>569</width>
        <height>757</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0,0,0,0,0,1,0,1,0,1">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QLabel" name="welcome_message_label_courseinfo">
         <property name="font">
          <font>
           <family>Noto Sans CJK HK</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Welcome [Name]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="course_title_label_courseinfo">
         <property name="font">
          <font>
           <family>Noto Sans CJK HK</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>[Course Title]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="announcement_label_courseinfo">
             <property name="font">
              <font>
               <family>Noto Sans CJK HK</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Announcement: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="latest_announcement_label_courseinfo">
             <property name="font">
              <font>
               <family>Noto Sans CJK HK</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>[Latest Announcement]</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="all_announcement_button_courseinfo">
           <property name="font">
            <font>
             <family>Noto Sans CJK HK</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>All Announcement</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="course_info_layout_courseinfo">
         <item>
          <widget class="QLabel" name="course_info_label_courseinfo">
           <property name="font">
            <font>
             <family>Noto Sans CJK HK</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Course Information:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="course_info_list_courseinfo">
           <item>
            <property name="text">
             <string>Time: [time]</string>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans CJK HK</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Classroom: [address]</string>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans CJK HK</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Zoom Link: [zoom link]</string>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans CJK HK</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Instructor: [instructor name]</string>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans CJK HK</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Instructor Email: [instructor email]</string>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans CJK HK</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="notes_layout_courseinfo">
         <item>
          <widget class="QLabel" name="notes_label_courseinfo">
           <property name="font">
            <font>
             <family>Noto Sans CJK HK</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Notes:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="notes_list_courseinfo">
           <item>
            <property name="text">
             <string>Lect1: [n_link1]</string>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans CJK HK</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lect2: [n_link2]</string>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans CJK HK</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="other_materials_label_courseinfo">
           <property name="font">
            <font>
             <family>Noto Sans CJK HK</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Other Materials:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="other_materials_list_courseinfo">
           <item>
            <property name="text">
             <string>Material1: [m_link1]</string>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans CJK HK</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Material2: [m_link2]</string>
            </property>
            <property name="font">
             <font>
              <family>Noto Sans CJK HK</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="email_layout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="email_button_courseinfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans CJK HK</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Email Me</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_13">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
